dev-tools/build_release.py

```diff
@@ -618,6 +610,60 @@ def check_norelease(path='src'):
           if pattern.search(line):
             raise RuntimeError('Found //norelease comment in %s line %s' % (full_path, line_number))
 
+def run_and_print(text, run_function):
+  try:
+    print(text, end='')
+    run_function()
+    print(OKGREEN + 'OK' + ENDC)
+    return True
+  except RuntimeError:
+    print(FAIL + 'NOT OK' + ENDC)
+    return False
+
+def check_env_var(text, env_var):
+  try:
+    print(text, end='')
+    env[env_var]
+    print(OKGREEN + 'OK' + ENDC)
+    return True
+  except KeyError:
+    print(FAIL + 'NOT OK' + ENDC)
+    return False
+
+def check_environment_and_commandline_tools(check_only):
+  checks = list()
```

Comment:
Why not []?

But also, why not just a boolean?  E.g. rename to `ok = True`, and then `ok = ok and check_env_var(..)` for each check?